core/nwb.behavior.yaml

```diff
@@ -38,6 +38,10 @@ groups:
         is 'meters'. Actual stored values are not necessarily stored in these units.
         To access the data in these units, multiply 'data' by 'conversion' and add 'offset'.
       required: false
+    - name: bounds
+      dtype: float32
+      doc: The boundary range for the data. The units are the same as the units for the data.
+      required: false
```

Comment:
```suggestion
    - name: bounds
      dtype: float32
      doc: The boundary range (min, max) for each dimension of `data`. The units are the same as the units for the data.
      shape:
      - - 1
        - 2
      - - 2
        - 2
      - - 3
        - 2   
      required: false
```

Reply:
Can you explain the min max part and how that relates to the dimensions? @rly

Reply:
If the data represent (x, y, z) positions over time, and x has min/max values -100 to 100, y has min/max values -10 to 10, and z has min/max values -1 to 1, then `bounds` would be [(-100, 100), (-10, 10), (-1, 1)], an array with shape (3, 2). Similarly, if data represents N dimensions of positions over time, then `bounds` would be an array with shape (N, 2).